.github/workflows/conan-build.yml

```diff
@@ -0,0 +1,558 @@
+          - build-type: Debug
+            cmake-preset: conan-debug
+          - build-type: Release
+            cmake-preset: conan-release
+
+    name: MSVC ${{ matrix.toolset }} - C++${{ matrix.cppstd }} - ${{ matrix.build-type }}
+    runs-on: windows-2025
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install VS 2019 Build Tools v142
+        if: matrix.toolset == 142
+        shell: powershell
+        run: |
+          if (-not (Get-Command choco -ErrorAction SilentlyContinue)) {
+            Set-ExecutionPolicy Bypass -Scope Process -Force
+            [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+            iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+          }
+          choco install -y visualstudio2019buildtools --package-parameters "--add Microsoft.VisualStudio.Workload.VCTools --add Microsoft.VisualStudio.Workload.NativeDesktop --add Microsoft.Component.MSBuild --add Microsoft.VisualStudio.Component.VC.v142.x86.x64 --add Microsoft.VisualStudio.Component.Windows10SDK.19041 --includeRecommended --installPath C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools"
+
+      - name: Install VS 2022 Build Tools v143
+        if: matrix.toolset == 143
+        shell: powershell
+        run: |
+          if (-not (Get-Command choco -ErrorAction SilentlyContinue)) {
+            Set-ExecutionPolicy Bypass -Scope Process -Force
+            [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+            iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+          }
+          choco install -y visualstudio2022buildtools --package-parameters "--add Microsoft.VisualStudio.Component.Windows10SDK.26100"
+
+      - name: Detect vcvarsall for VS 2019 Build Tools
+        if: matrix.toolset == 142
+        uses: ilammy/msvc-dev-cmd@v1.13.0
+        with:
+          arch: amd64
+          vsversion: 2019
+          sdk: 10.0.19041.0
+
+      - name: Detect vcvarsall for VS 2022 Build Tools
+        if: matrix.toolset == 143
+        uses: ilammy/msvc-dev-cmd@v1.13.0
+        with:
+          arch: amd64
+          vsversion: 2022
+          sdk: 10.0.26100.0
+
+      - name: Install CMake
+        uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 4.1.0
+          ninjaVersion: latest
+
+      - name: Install Conan
+        uses: turtlebrowser/get-conan@main
+
+      - name: Conan select default profile
+        shell: powershell
+        run: |
+          choco install -y ninja
+          conan --version
+          conan profile detect
+
+      - name: Install dependencies
+        if: matrix.toolset == 142
+        run: >
+          conan install .
+          -c tools.microsoft.msbuild:vs_version=16
+          -c tools.microsoft:winsdk_version=10.0.19041.0
```

Comment:
Is this essential to hardcode windows sdk version? Shall we keep eye on it when it would be deprecated and unsupported completely?